draft-ietf-rtcweb-jsep.xml

```diff
@@ -3929,13 +3920,16 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
             that the JSEP implementation should be sending media
             ("sendonly" for local answers, "recvonly" for remote
             answers, or "sendrecv" for either type of answer),
-            choose the media format to send
-            as the most preferred media format from the remote
-            description that is also present in the answer, as
-            described in
-            <xref target="RFC3264" />, Sections 6.1 and 7, and start
-            transmitting RTP media once the underlying transport layers
-            have been established. If an SSRC has not already been
+            choose the media format to send; for the offerer, this will be
+            the most preferred media format from the answer that is also
+            locally supported, as described in <xref target="RFC3264" />,
+            Section 7; for the answerer, this will be
```

Comment:
Well, this gets at an interesting point. 3264 says that the offerer has to pick something from the answer, but the answerer has to pick something that is in both. So there is an edge case where the offerer offers A B due to codec prefs, and the answerer replies with C A B; is it legal for the offerer to send with C?

It seems illegal for you to send with something you haven't offered, and that would clean up a lot of things (including CN handling), but I'm not positive this is always true. @adamroach, any thoughts on this?